connect/eaas/worker.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, secure=True):
         self.extension_type = get_extension_type(self.extension_class)
         descriptor = self.extension_class.get_descriptor()
         self.capabilities = descriptor['capabilities']
+        self.variables = descriptor['variables'] if 'variables' in descriptor else []
```

Comment:
Here you should do just descriptor.get('variables') because None is supported on both sides and this ensure backward compatibility on the backend side